DataProducts/inc/STMTestBeamEventInfo.hh

```diff
@@ -0,0 +1,22 @@
+#ifndef DataProducts_STMTestBeamEventInfo_hh
+#define DataProducts_STMTestBeamEventInfo_hh
+
+namespace mu2e {
+  class STMTestBeamEventInfo {
+  public:
+    enum TrigType { unknown=-1, internal=0, external=1 };
+    STMTestBeamEventInfo() : _trigType(TrigType::unknown), _triggerTime(0) { }
+
+    void trigType(uint16_t type) { _trigType = TrigType(type);}
+    void triggerTime(uint64_t time) { _triggerTime = time;}
+
```

Comment:
When you create an object of this type, are the two data members known at c'tor time?  If so, please provide a c'tor that takes both of these as arguments.   In general classes should only have setter functions if it's not possible to construct the object completely when the c'tor is called.

Reply:
I move a few things in the source module to create the object when we know what its contents will be (6d6f873)